configs/sst_platform_tools-compat.yaml

```diff
@@ -1,41 +0,0 @@
-document: feedback-pipeline-workload
-version: 1
-data:
-  name: Toolchain compatibility packages
-  description: >
-    Collection of packages providing compatibility packages related to
-    the GNU Toolchain.
-  maintainer: sst_platform_tools
-  
-  packages: []
-
-  package_placeholders:
-    # RHEL 8 bumped SONAME of libgfortran.so.  Support RHEL 7 apps.
-    compat-libgfortran-48:
-      description: RHEL-only compat package.
-      srpm: compat-libgfortran-48
```

Comment:
This package is still needed in RHEL 9, correct?  If so, you don't want to delete this file.  You just want to remove the `eln` label from it.

Reply:
That is correct. I have restored the file and just removed the eln label. Thanks @jwboyer